test/E2E/Browser.purs

```diff
@@ -0,0 +1,100 @@
+module Test.E2E.Browser
+  ( Mode(..)
+  , TestOptions(..)
+  , launchWithExtension
+  , parseOptions
+  ) where
+
+import Prelude
+
+import Control.Applicative (apply, (<$>))
+import Data.Foldable (fold)
+import Data.Maybe (Maybe(Just, Nothing), fromMaybe)
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Effect.Class (liftEffect)
+import Effect.Console (error)
+import Options.Applicative
+  ( Parser
+  , execParser
+  , help
+  , long
+  , metavar
+  , option
+  , showDefault
+  , str
+  , strOption
+  , switch
+  , value
+  , info
+  , fullDesc
+  )
+import Toppokki as Toki
+
+data TestOptions = TestOptions
```

Comment:
```suggestion
newtype TestOptions = TestOptions
```

Reply:
You should derive instances for this as well (`Eq`, `Newtype`, `Generic`) and add `Show` with `genericShow`